react-d3-chart/src/components/Bisector/BisectorTooltip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import { BisectorTooltipEntry } from '../../types';
+
+const TooltipContainer = styled.rect`
+  fill: #e8e8e8;
+`;
+
+export interface Props {
+  data: Array<BisectorTooltipEntry>;
+  x: number;
+  width: number;
+  height: number;
+  marginLeft: number;
+}
+
+const BisectorTooltip: React.FC<Props> = ({
+  data,
+  x,
+  width,
+  height,
+  marginLeft,
+}) => {
+  const tooltipEntries = data.map((entry, index) => (
+    <tspan x={5} dy="14px" fill={entry.colour} key={`entry${index}`}>
+      {`Trace ${index + 1}: ${entry.y}`}
+    </tspan>
+  ));
+
+  return (
+    <g transform={`translate(${marginLeft + 20}, 0)`}>
+      <TooltipContainer width={width} height={height} />
+      <text>
+        <tspan x={5} y={5} dy="13px" fontWeight="bold">
```

Comment:
what are these `x={5} y={5} ` hardcoded values?

Reply:
This is the offset (margin) of the tooltip text inside the rect 
![image](https://user-images.githubusercontent.com/22064715/88672007-f61ce180-d0b4-11ea-9d1d-efe78f44ac68.png)

Reply:
Did you use d3 color scheme? The coloring here looks a bit off (e.g. trace 1 and 2 look too light) . I remember the first one of d3 color is always steelblue then orange.